src/publish-search.js

```diff
@@ -1,12 +1,16 @@
 'use strict';
 
+const btoa = require('btoa');
 const fs = require('fs-extra');
 const request = require('request');
 const path = require('path');
-const endpoint = process.env.searchEndpoint;
-const token = process.env.token;
-const filePath = path.join(process.cwd(), 'src', 'stache', 'search', 'search.json');
+const audienceId = process.env.audienceId;
+const clientUserName = process.env.clientUserName;
+const clientKey = process.env.clientKey;
+const endpoint = process.env.endpoint;
```

Comment:
Non-blocking, but the following format might be easier to modify/read later on.

```
const {
  audienceId,
  clientUserName,
  clientKey,
  endpoint
} = process.env;
```

Reply:
Good call! I forget about destructuring sometimes.